content/automate/ManageIQ/Infrastructure/VM/Provisioning/StateMachines/ProvisionRequestApproval.class/__methods__/validate_request.rb

```diff
@@ -113,7 +113,8 @@
 # Use value from model unless specified above
 max_memory ||= $evm.object['max_memory']
 unless max_memory.nil?
-  $evm.log("info", "Auto-Approval Threshold(Model):<max_memory=#{max_memory}> detected")
+  max_memory = max_memory.to_i.megabytes
```

Comment:
@billfitzgerald0120 - Same question as [here](https://github.com/ManageIQ/manageiq-content/pull/273/files#r202179618) if we can always trust that all infrastructure providers use the same base numbers for `max_memory`

Reply:
@syncrou The value will be consistent here. The provider values could differ in the base numbers for the requested values.